Code/Bench/descriptors.cpp

```diff
@@ -0,0 +1,31 @@
+#include <catch2/catch_all.hpp>
+#include <string>
+
+#include "bench_common.hpp"
+
+#include <GraphMol/SmilesParse/SmilesParse.h>
+#include <GraphMol/Descriptors/Lipinski.h>
+
+using namespace RDKit;
+
+TEST_CASE("Descriptors::calcNumSpiroAtoms", "[descriptors]") {
+  auto samples = bench_common::load_samples();
+  BENCHMARK("Descriptors::calcNumSpiroAtoms") {
+    auto sum = 0;
+    for (auto &mol : samples) {
+      sum += Descriptors::calcNumSpiroAtoms(mol);
+    }
+    return sum;
```

Comment:
You'll see this pattern of taking some sum, then returning it. It's a precaution against the optimizer. Prevents code from being optimized away. Sometimes it's necessary but it's hard to tell when.

Catch2 does a pretty good job detecting optimized-away functions, it emits an error along the lines of `Benchmark failed (could not measure benchmark, maybe it was optimized away)`